.gitignore

```diff
@@ -20,4 +20,5 @@ apk-sizes-*.txt
 *.binlog
 src/Xamarin.Android.Build.Tasks/Xamarin.Android.Common.props
 *~
+external/mono/
```

Comment:
kinda surprised that this PR doesn't need rebasing, but this change is included in 83ea4faeb8c651a97cc54307a4d065d8275a1e88, and thus is not needed anymore.